pkg/receiver/rawk8seventsreceiver/README.md

```diff
@@ -38,4 +38,37 @@ receivers:
 The full list of settings exposed for this receiver are documented in
 [config.go](./config.go).
 
+## Persistent Storage
+
+If a storage extension is configured in the collector configuration's `service.extensions` property,
+the `raw_k8s_events` receiver stores the latest resource version retrieved from the Kubernetes API server
+so that after a restart, the receiver will continue retrieving events starting from that resource version
+instead of using the `max_event_age` property.
+This prevents the receiver from reporting duplicate events when the receiver is restarted in less than the `max_event_age` time.
+On the other hand, this also allows the receiver to catch up on missed events in case it was not running for longer than `max_event_age` time.
+Note that the default maximum age of events retained by the API Server is one hour - see the [--event-ttl][event_ttl] option of `kube-apiserver`.
+
+Example configuration:
+
+```yaml
+extensions:
+  file_storage:
+    directory: .
+
+receivers:
+  raw_k8s_events:
+
+service:
+  extensions:
+  - file_storage
+  pipelines:
+    logs:
+      receivers:
+      - raw_k8s_events
+      exporters:
+      - nop
+```
+
 [Fluentd plugin]: https://github.com/SumoLogic/sumologic-kubernetes-fluentd/tree/main/fluent-plugin-events
+
+[event_ttl]: https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/#options
```

Comment:
```suggestion
[event_ttl]: https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/#options
```

Reply:
I'm sorry I'm not seeing any change here, what are you suggesting?

Reply:
Oh it's the empty line 😄 I see now

Reply:
Done